server/controllers/dishes/updateDish.ts

```diff
@@ -0,0 +1,33 @@
+import { Request, Response, NextFunction } from 'express';
+import { Dish } from '../../models';
+import { dishSchema } from '../../utils/validation/joi';
+import { CustomError } from '../../utils';
+import { StatusCodes } from '../../utils/enum';
+
+const updateDish = async (req: Request, res: Response, next: NextFunction) => {
+  const { id } = req.params;
```

Comment:
Destructure the id in the try statement's scoop. Think what will happen when the user doesn't send an id in the params.

Reply:
done 👍